test/transform.test.ts

```diff
@@ -326,6 +326,62 @@ describe("transform (jsdoc)", () => {
     });
   });
 
+  it("does not split example without codeblock", () => {
+    const result = transform(`
+      export default {
+        /**
+         * @note This is a note.
+         * @example
+         * export default secretNumber = 42
+         *
+         * export default nothing = null
+         * @type {'src' | 'root'}
+         */
+        srcDir: 'src'
+      }
+    `);
+    expectCodeToMatch(result, /export default ([\S\s]*)$/, {
+      srcDir: {
+        $default: "src",
+        $schema: {
+          title: "",
+          tsType: "'src' | 'root'",
+          description: "",
+          tags: [
+            "@note This is a note.",
+            "@example\nexport default secretNumber = 42\n\nexport default nothing = null",
+          ],
+        },
+      },
+    });
+  });
+
+  it("supports codeblock without `@example`", () => {
+    const result = transform(`
+      export default {
+        /**
+         * @note This is a note.
+         * \`\`\`js
+         * export default secret
+         * \`\`\`
+         * @type {'src' | 'root'}
+         */
+        srcDir: 'src'
+      }
+    `);
+    expectCodeToMatch(result, /export default ([\S\s]*)$/, {
+      srcDir: {
+        $default: "src",
+        $schema: {
+          title: "",
+          tsType: "'src' | 'root'",
+          description: "",
+          tags: ["@note This is a note.", "```js\nexport default secret\n```"],
```

Comment:
Shouldn't inline code go into the description?

Reply:
That's a good question. I don't know. What do you think? 🤷‍♂️

Reply:
But because there is the `@`, the title and description are the content above

see https://github.com/unjs/untyped/blob/main/src/loader/babel.ts#L253